src/officehours_api/backends/zoom.py

```diff
@@ -141,17 +141,25 @@ def _get_client(cls, user: User) -> ZoomClient:
     def _create_meeting(cls, user: User) -> ZoomMeeting:
         """Creates a Zoom meeting for the given user."""
         client = cls._get_client(user)
-        user_id = user.profile.backend_metadata['zoom']['user_id']
-        # Specify the meeting settings
-        # We are initializing the meeting settings with the default settings
-        # and then modifying the settings we want to change
-        meeting_settings = ZoomMeetingSettings.default_settings()
-        meeting_settings.waiting_room = True
-        meeting_settings.join_before_host = False
-        meeting_settings.meeting_authentication = False
-        meeting_settings.use_pmi = False
-        meeting_settings.contact_name = user.first_name + ' ' + user.last_name
-        meeting_settings.contact_email = user.email
+        meeting_settings = ZoomMeetingSettings(
+            approval_type=0,
```

Comment:
Okay, I compared this to the previous settings and [zoom defaults](https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/meetingCreate)

This should be changed and gets it in sync. I think everything else looks fine.

```suggestion
            approval_type=2,
```

Reply:
Thanks!  The documentation seems to indicate that would resolve the registration issue, too.